app/_locales/en/messages.json

```diff
@@ -1837,6 +1838,12 @@
   "functionType": {
     "message": "Function type"
   },
+  "fundYourWallet": {
+    "message": "Fund your wallet"
+  },
+  "fundYourWalletDescription": {
+    "message": "Get started by adding some $1 to your wallet."
```

Comment:
let's add the same description of `"buyToken"` here (I'm assuming `$1` is the symbol)